hummingbot/connector/exchange/ascend_ex/ascend_ex_exchange.py

```diff
@@ -526,8 +520,8 @@ async def _create_order(self,
             )
             resp = await self._api_request(
                 method="post",
-                path_url="cash/order",
-                params=api_params,
+                path_url=CONSTANTS.ORDER_PATH_URL,
+                data=api_params,
```

Comment:
Do verify if using the body parameters is correct for this particular endpoint.

Reply:
Reason for change for this one was because it was initially erroring-out, but double-checked and confirmed.